android/src/main/java/nordpol/TagDispatcher.java

```diff
@@ -153,7 +158,52 @@ private void disableForegroundDispatch(NfcAdapter adapter) {
         adapter.disableForegroundDispatch(activity);
     }
 
+    static byte[] requireStatus(byte[] receivedApdu, byte[] expectedApdu) throws IOException {
```

Comment:
@Franzaine I can see that having this function  returning the APDU may be helpful, but I think having it return a boolean would be more universally useful. It is not always an error that the status code does not match. What do you think?